### PR TITLE
Only end coverage if there is actually a proc running

### PR DESCRIPTION
**Pull request by imurchie**

### CODE REVIEW

File: lib/tools/adb-commands.js [hunk 1]
```diff
@@ -1071,7 +1071,7 @@ methods.broadcast = async function (intent) {
  * Kill Android instruments if they are currently running.
  */
 methods.endAndroidCoverage = async function () {
-  if (this.instrumentProc) {
+  if (this.instrumentProc && this.instrumentProc.isRunning) {
```
Comment (mykola-mokhnach) [inline lib/tools/adb-commands.js]: is it a property?

Comment (imurchie) [reply lib/tools/adb-commands.js]: It's a getter.

Comment (imurchie) [reply lib/tools/adb-commands.js]: https://github.com/appium/node-teen_process/blob/master/lib/teen_process.js#L131-L134

**Comment by imurchie** [comment]
The only time I've seen this actually happening is when the process is never started (when we have a failure on session start and never actually get to the start coverage stage).

**Pull request merged**